modal/io_streams.py

```diff
@@ -184,7 +184,7 @@ async def _consume_container_process_stream(self):
 
                 async for message in iterator:
                     if self._stream_type == StreamType.STDOUT and message:
-                        print(message, end="")
+                        print(message.decode("utf-8"), end="")
```

Comment:
should we condition this on `self._text`? it probably makes sense for the behavior to differ passing in `text=True` vs. `text=False` to `exec`

Reply:
Hm I'm not sure. When would you want to write raw bytes to stdout?

Reply:
good point—`subprocess.Popen` seems to only use the `text` arg when stdout/stderr are piped, so always sending text here seems more consistent